server/dummyData.js

```diff
@@ -26,315 +26,327 @@ module.exports = function() {
-      //userTwo image block
-      imagelicenseU1.fileHandle = "userTwo";
-      imagelicenseU1.extension = "jpg";
-      imagelicenseU1.save({}).catch(function(err) {
+      let imagelicenseU2 = new Image();
+      //user2 image block
+      imagelicenseU2.fileHandle = "user2";
+      imagelicenseU2.extension = "jpg";
+      imagelicenseU2.save({}).catch(function(err) {
         if (11000 === err.code || 11001 === err.code) {
-          console.log("DB Error: Duplicate of license:", imagelicenseU1.fileHandle);
+          console.log("DB Error: Duplicate of license:", imagelicenseU2.fileHandle);
         }
       });
-      licenseU1.image = imagelicenseU1;
-      licenseU1.save({}).catch();
-      userTwo.license = licenseU1;
+      licenseU2.image = imagelicenseU2;
+      licenseU2.save({}).catch();
+      user2.license = licenseU2;
     
-      let addressU1 = new Address();
-      //userTwo address block
-      addressU1.street1 = "1 Dickson place";
-      addressU1.street2 = "";
-      (addressU1.suburb = "Dickson"), (addressU1.state = "ACT");
-      addressU1.postCode = "2602";
-      addressU1.save({}).catch();
-      userTwo.address = addressU1;
+      let addressU2 = new Address();
+      //user2 address block
+      addressU2.street1 = "1 Dickson place";
+      addressU2.street2 = "";
+      (addressU2.suburb = "Dickson"), (addressU2.state = "ACT");
+      addressU2.postCode = "2602";
+      addressU2.save({}).catch();
+      user2.address = addressU2;
     
-      let creditCardU1 = new CreditCard();
-      //userTwo address block
-      creditCardU1.cardNumber = " 3778 123456 12345";
-      creditCardU1.nameOnCard = "userTwo";
-      creditCardU1.ccv = "3547";
-      creditCardU1.expiryMonth = 12;
-      creditCardU1.expiryYear = 2020;
-      creditCardU1.save({}).catch();
-      userTwo.creditCard = creditCardU1;
+      let creditCardU2 = new CreditCard();
+      //user2 address block
+      creditCardU2.cardNumber = " 3778 123456 12345";
+      creditCardU2.nameOnCard = "user2";
+      creditCardU2.ccv = "3547";
+      creditCardU2.expiryMonth = 12;
+      creditCardU2.expiryYear = 2020;
+      creditCardU2.save({}).catch();
+      user2.creditCard = creditCardU2;
     
-      let confirmationCodeU1 = new ConfirmationCode();
-      confirmationCodeU1.code = "ABC123456789";
-      confirmationCodeU1.user = userTwo;
-      confirmationCodeU1.expiresAt = "2018-05-01";
-      confirmationCodeU1.save({}).catch();
-      userTwo.confirmationCodes = confirmationCodeU1;
+      let confirmationCodeU2 = new ConfirmationCode();
+      confirmationCodeU2.code = "ABC123456789";
+      confirmationCodeU2.user = user2;
+      confirmationCodeU2.expiresAt = "2018-05-01";
+      confirmationCodeU2.save({}).catch();
+      user2.confirmationCodes = confirmationCodeU2;
     
-      userTwo.save({}).catch(function(err) {
+      user2.save({}).catch(function(err) {
         if (11000 === err.code || 11001 === err.code) {
```

Comment:
`11000 === err.code` can we move the 11000 into a variable like `const uniqueViolation = 11000` and `if(uniqueViolation === err.code)`

Reply:
Sorry forgot to add this one lol